db/migrate/20170609101302_drop_public_body_translated_columns.rb

```diff
@@ -0,0 +1,35 @@
+# -*- encoding : utf-8 -*-
+class DropPublicBodyTranslatedColumns < ActiveRecord::Migration
+  def up
+    remove_column :public_bodies, :name
+    remove_column :public_bodies, :short_name
+    remove_column :public_bodies, :request_email
+    remove_column :public_bodies, :url_name
+    remove_column :public_bodies, :notes
+    remove_column :public_bodies, :first_letter
+    remove_column :public_bodies, :publication_scheme
+  end
+
+  def down
+    add_column :public_bodies, :name, :text
+    add_column :public_bodies, :short_name, :text
+    add_column :public_bodies, :request_email, :text
+    add_column :public_bodies, :url_name, :text
+    add_column :public_bodies, :notes, :text
+    add_column :public_bodies, :first_letter, :text
```

Comment:
Some of these have default values and constraints that need to get reapplied. Also check types – some might be `:string` rather than `:text`